### ISSUE TITLE
GUI module collision with electrum-ltc

### ISSUE THREAD

**Issue by wilsonehusin** [vertcoin/electrum-vtc#16]
Initially I had electrum-ltc and things are working fine. After I install electrum-vtc, both will not launch and it seems that some module are mixed up between the two. Maybe someone can shed a light?
```
$  electrum-ltc
Traceback (most recent call last):
  File "/usr/local/bin/electrum-ltc", line 379, in <module>
    d.init_gui(config, plugins)
  File "/usr/local/lib/python2.7/dist-packages/Electrum_VTC-2.8.3.2-py2.7.egg/electrum_ltc/daemon.py", line 281, in init_gui
    gui = __import__('electrum_ltc_gui.' + gui_name, fromlist=['electrum_ltc_gui'])
ImportError: No module named vtc
$  electrum-vtc
Traceback (most recent call last):
  File "/usr/local/bin/electrum-vtc", line 4, in <module>
    __import__('pkg_resources').run_script('Electrum-VTC==2.8.3.2', 'electrum-vtc')
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 719, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1504, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/Electrum_VTC-2.8.3.2-py2.7.egg/EGG-INFO/scripts/electrum-vtc", line 379, in <module>
    d.init_gui(config, plugins)
  File "/usr/local/lib/python2.7/dist-packages/Electrum_VTC-2.8.3.2-py2.7.egg/electrum_ltc/daemon.py", line 281, in init_gui
    gui = __import__('electrum_ltc_gui.' + gui_name, fromlist=['electrum_ltc_gui'])
ImportError: No module named vtc
```

**Comment by metalicjames** [comment]
Potentially fixed in 827a38292fa95759a02779a076a7106c629dbfed. Let me know if it works for you.

**Comment by wilsonehusin** [comment]
Fixed! If you had the same issue, reinstalling may not make it work right away. I had to uninstall both electrum-ltc and electrum-vtc, then install both.

**Comment by wilsonehusin** [comment]
Looks like now nothing is crashing, but electrum-ltc will open electrum-vtc. If I uninstall and reinstall electrum-ltc, both commands will open electrum-ltc.

TL;DR both links now open whichever is installed later
